collect_app/src/test/java/org/odk/collect/android/formmanagement/metadata/FormMetadataParserTest.kt

```diff
@@ -171,6 +172,7 @@ class FormMetadataParserTest {
                                 action="foo" 
                                 orx:auto-send="bar" 
                                 orx:auto-delete="baz" 
+                                orx:client-editable="qux" 
```

Comment:
`client-editable` uses the `odk` namespace: https://forum.getodk.org/t/spec-proposal-editable-submissions/53577/8.

We should also make sure to add a test that `client-editable` using a different name space is not parsed (it looks like the implementation is namespace agnostic right now).

Reply:
Fixed.